library/ssl_msg.c

```diff
@@ -5171,7 +5169,7 @@ static int ssl_handle_hs_message_post_handshake( mbedtls_ssl_context *ssl )
         MBEDTLS_SSL_DEBUG_MSG( 3, ( "refusing renegotiation, sending alert" ) );
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
-        if( ssl->minor_ver >= MBEDTLS_SSL_MINOR_VERSION_1 )
+        if( ssl->minor_ver == MBEDTLS_SSL_MINOR_VERSION_3 )
```

Comment:
`>=`

Reply:
Thanks for finding that - I completely forgot about the TLS 1.3. Code fixed and pushed.